### DISCUSSION TITLE
Send to print and export PDF the sum of a column in footer.

### DISCUSSION THREAD

**Discussion by DannyV90** [yajra/laravel-datatables#2636] [Q&A]
Hi to everybody,
I'm using datatables as service in laravel.
I found how add to the footer of a column the sum of that field getting it from serverside.

```

->withQuery('sum', function($filteredQuery) {
                return $filteredQuery->sum('price');
            })

```

Now I would want print it in footer also in the export files (PDF and Print).

I'm using snappy as generator. I tried dd($data) in print.blade.php view, but only columns data was sent.

Somebody can tell me how sending also sums in an array with keys (like $data for columns) to the view so I can place them in the right column of the footer?

Thanks in advance

**Comment by yajra** [comment]
Override this property: `protected $printPreview = 'path.to.custom.view';`. 

See `datatables::print.blade.php` for the original contents as the template and adjust as needed.

**Reply by DannyV90** [reply]
Thank you very much yajra!
